storage/sqlStorages/postgres/datasource_test.go

```diff
@@ -29,7 +29,7 @@ func TestDataSource_Get(t *testing.T) {
 
 	db, err := sql.Open("postgres", psqlInfo)
 	if err != nil {
-		panic("Couldn't connect to a database")
+		panic("Couldn'template connect to a database")
```

Comment:
emm... I guess some refactor didn't go as planned :D

Reply:
It's actually in every line changed in this file (don't want to overwhelm you with lots of comments)